dali/pipeline/data/tensor_list.h

```diff
@@ -646,24 +646,24 @@ class DLL_PUBLIC TensorList {
     return {data_.capacity()};
   }
 
-  /**
-   * @brief Set the type of the TensorList. The type needs to be set before calling
-   * the Resize function that gives the shape. Type can be changed, if the current storage
-   * is not big enough, the memory will be reallocated.
+  /** @defgroup SetTypeFunctions Type setting functions.
```

Comment:
I'm terribly sorry to disappoint you, but this doesn't work - visit the generated documentation page and see that `set_type` is now totally undocumented.

Reply:
Fixed, should be ok now